app/helpers/navigation_items.rb

```diff
@@ -58,9 +58,7 @@ def for_provider_primary_nav(current_provider_user, current_controller, performi
           items << NavigationItem.new('Activity log', provider_interface_activity_log_path, active?(current_controller, %w[activity_log]))
         end
 
-        if FeatureFlag.active?(:export_application_data) || FeatureFlag.active?(:export_hesa_data)
-          items << NavigationItem.new('Export data', provider_interface_new_application_data_export_path, active?(current_controller, %w[application_data_export hesa_export]), 'app-primary-navigation__item--align-right')
-        end
+        items << NavigationItem.new('Reports', provider_interface_reports_path, active?(current_controller, %w[reports application_data_export hesa_export]), 'app-primary-navigation__item--align-right')
```

Comment:
Is the plan to always show the reports link now?

Reply:
Yes